src/core/server/graph/tenant/schema/schema.graphql

```diff
@@ -232,7 +232,10 @@ type PageInfo {
   """
   Indicates that there are more nodes after this subset.
   """
-  hasNextPage: Boolean!
+  hasNextPage: Boolean
+  hasPreviousPage: Boolean
```

Comment:
Should remove reverse pagination arguments as the server does not support it.

Reply:
I can't, relay needs it to be defined 🤷‍♂️

Reply:
I thought we were able to handle that in the network layer?

Reply:
Nope, the compiler is complaining. We did error handling at the network layer.